package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Install the latest go-ipfs binary",
   "main": "src/index.js",
   "scripts": {
-    "install": "node src/bin.js",
+    "install": "node src/bin.js $TARGET_VERSION $TARGET_OS",
```

Comment:
This will not work on windows, but there is https://www.npmjs.com/package/cross-env to fix that